README.md

```diff
@@ -91,10 +91,10 @@ available Cloud Run regions](https://cloud.google.com/run/docs/locations))
 
 The rollout strategy consists of the steps and health criteria.
 
-- `-cli-run-interval`: The time between each health check, in seconds (default:
-`60`). This is only need it if running with `-cli` option.
-- `-healthcheck-offset`: To evaluate the candidate's health, use metrics from
-the last `N` minutes relative to current rollout process (default: `30`)
+- `-cli-run-interval`: The time between each health check (default: `60s`). This
+is only need it if running with `-cli` option.
+- `-healthcheck-offset`:range of time for retrieving metrics relative to current
```

Comment:
I'd probably rewrite this as 

> Duration to look back for metrics to assess health of the candidate revision.

or

> time window to look back at the time of health check to assess health of the candidate revision.

Reply:
I like second more but would rephrase it to

> time window to look back during health check to assess the candidate revision's health.

to avoid the repetition of the word "time" in different contexts and to separate the occurrences of "health" a little bit more.